src/metaswitch/clearwater/etcd_tests/etcdserver.py

```diff
@@ -8,7 +8,7 @@
 from shutil import rmtree
 import uuid
 
-base_cmd =              """clearwater-etcd/usr/bin/etcd --debug --listen-client-urls http://{0}:4000 --advertise-client-urls http://{0}:4000 --listen-peer-urls http://{0}:2380 --initial-advertise-peer-urls http://{0}:2380 --data-dir {2} --name {1}"""
+base_cmd =              """clearwater-etcd/usr/share/clearwater/clearwater-etcd/3.1.7/etcd --debug --listen-client-urls http://{0}:4000 --advertise-client-urls http://{0}:4000 --listen-peer-urls http://{0}:2380 --initial-advertise-peer-urls http://{0}:2380 --data-dir {2} --name {1}"""
```

Comment:
We've discussed, and I'm going to handle this in a separate PR that looks at some other issues with the etcd tests.